### DISCUSSION TITLE
can bband-adaband be used?

### DISCUSSION THREAD

**Discussion by poadb** [staxrip/staxrip#736] [Q&A] [answered]
https://github.com/vztu/bband-adaband

Looks very promising for banding detection and debanding.
It's written in MATLAB

**Comment by Dendraspis** [comment, marked as answer]
StaxRip uses AviSynth+ and VapourSynth as Frameservers, that use their file extensions - as far as I remember written in Python or so.
So in order to use Matlab code you need to re-write it or use a filter, that can load the code.

I'm not very into filters, but I would summerize it with  ***No***.

**Reply by JJKylee** [reply]
@Dendraspis is right. The MATLAB code should be translated to AviSynth or VapourSynth.
